resources/assets/coffee/react/beatmapset-page/header.coffee

```diff
@@ -142,17 +147,19 @@ export class Header extends React.Component
 
           el BeatmapsetMapping, beatmapset: @props.beatmapset
 
-          if currentUser.id? && @props.beatmapset.availability
+          if currentUser.id? && @hasAvailabilityInfo()
             div
               className: 'beatmapset-header__availability-info',
               if @props.beatmapset.availability.download_disabled
                 osu.trans 'beatmapsets.availability.disabled'
-              else
-                osu.trans 'beatmapsets.availability.parts-removed'
-
-              if @props.beatmapset.availability.more_information
-                div className: 'beatmapset-header__availability-link',
-                  a href: @props.beatmapset.availability.more_information, target: '_blank', osu.trans 'beatmapsets.availability.more-info'
+              else if @props.beatmapset.availability.more_information?
```

Comment:
This be backwards - should be checking if `.more_information` exists and falling back to the default message only if it's missing.